integrations/gcp/main.py

```diff
@@ -180,8 +185,19 @@ async def feed_events_callback(request: Request) -> Response:
         logger.info(
             f"Got Real-Time event for kind: {asset_type} with name: {asset_name} from project: {asset_project}"
         )
+        resource_configs = typing.cast(
+            GCPPortAppConfig, event.port_app_config
+        ).resources
+        matching_resource_configs = [
+            resource_config
+            for resource_config in resource_configs
+            if (
+                resource_config.kind == asset_type
+                and isinstance(resource_config.selector, GCPResourceSelector)
+            )
+        ]
         asset_resource_data = await feed_event_to_resource(
-            asset_type, asset_name, asset_project, asset_data
+            asset_type, asset_name, asset_project, asset_data, matching_resource_configs
```

Comment:
Iterate over matching_resource_config to register `asset_resource_data` for each kind that is a match